modules/apps/layout/layout-test/src/testIntegration/java/com/liferay/layout/workflow/handler/test/LayoutWorkflowHandlerTest.java

```diff
@@ -108,6 +111,17 @@ public void setUp() throws Exception {
 
 	@After
 	public void tearDown() throws Exception {
+		List<WorkflowInstance> workflowInstances =
+			WorkflowInstanceManagerUtil.getWorkflowInstances(
+				TestPropsValues.getCompanyId(), TestPropsValues.getUserId(),
+				new String[] {Layout.class.getName()}, false, -1, -1, null);
```

Comment:
Hi @seltong,

just replace `-1, -1` to `QueryUtil.ALL_POS, QueryUtil.ALL_POS`

Thank you!